xpublish/utils/zarr.py

```diff
@@ -104,10 +108,11 @@ def create_zmetadata(dataset):
     zmeta['metadata'][group_meta_key] = {'zarr_format': zarr_format}
     zmeta['metadata'][attrs_key] = _extract_dataset_zattrs(dataset)
 
-    for key, da in dataset.variables.items():
+    for key in dataset.variables.keys():
+        da = dataset[key]
```

Comment:
This is necessary instead of the iterator because we need a `DataArray` in order to get the `coordinates`. If we used `ds.data_vars.items()` coordinate variables are not included.